ui/wallet_settings_page.go

```diff
@@ -67,6 +67,13 @@ func (pg *walletSettingsPage) Layout(gtx layout.Context, common pageCommon) layo
 				common.changePage(PageWallet)
 			},
 			body: func(gtx layout.Context) layout.Dimensions {
+				if common.info.Wallets[*common.selectedWallet].IsWatchingOnly {
```

Comment:
This looks like a repetition and it could be handled with an if statement inside the rigid layout. Something like this 
`layout.Rigid(`
						`func(gtx layout.Context) layout.Dimensions {`
							`  if !common.info.Wallets[*common.selectedWallet].IsWatchingOnly {`
								`return pg.changePassphrase()(gtx)`
							`}`
							`return layout.Dimensions{}`
						`}),`
					`layout.Rigid(pg.notification()),`
					`layout.Rigid(pg.debug()),`
					`layout.Rigid(pg.dangerZone()),`